### DISCUSSION TITLE
Deploy TailwindCss into a Fastapi app on deta micro

### DISCUSSION THREAD

**Discussion by opticrew** [deta/cloud-docs#562] [Help]
Do you know how deploy to tailwindcss in a fastapi app serving Ninja html templates on deta ?

Everything works very well on my local computer thanks to the tailwindcss cli.

Once deployed on deta, everything works except the tailwindcss style are not applied.

Can someone help me.

Thanks in advance.

Antoine

**Comment by tbdsux** [comment]
You need to build your TailwindCss style and link it to your templates before deploying it.

**Reply by opticrew** [reply]
I think I've already build into my project and link it to my templates because it works when I run my fastapi app locally on my comptuter.

But when I deploy in production to deta, everything works except tailwind.

I think I miss something but I don't know what.

**Comment by nghiepdev** [comment]
You can use `tailwindcss` via CDN but it is not recommended for production.
